### DISCUSSION TITLE
github-tags with different packagenames interpolated in the tagname.

### DISCUSSION THREAD

**Discussion by maartenvanderhoef** [renovatebot/renovate#15463] [Request Help]
Hi everyone,

I have a mono repository with different tags for every individual version of folders within the repository, this results in different tags within the repository like so. The github repository is org/repo

```
label/v1.0.0
label/v1.1.0
foo/v1.0.0
foo/v2.0.0
foo/v2.1.0
bar/v1.0.0
```
The modules correspond to a zipfile stored on AWS S3, which are the artifacts I am actually using..  Terraform files which then source terraform s3 source which is supported by it's own authentication mechanism.

```
"s3::https://s3-eu-central-1.amazonaws.com/xx-module-releases/module-label-v1.0.0.zip"
```

With the regexmanager I can capture the version name  v1.0.0, the depname label. But with the datasource github-tags the packageName should always be `org/repo` for the datasource to work, and I do not see a way to use the match 'label' , with the tag 'label/v.1.0.0'. I don't see a way to filter on tags specifically. Is there something I can do ?

**Comment by rarkins** [comment]
Check if extractVersion can satisfy your needs.
